### PR TITLE
chore(samples): fix Vue samples by shifting to vite

### PR DESCRIPTION
**Pull request by louis-bompart**
https://coveord.atlassian.net/browse/KIT-2991

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -277,6 +277,7 @@
     "Unsubscriber",
     "uparrow",
     "upvoted",
+    "Vetur",
```
Comment (louis-bompart) [inline .cspell.json:280]:
```suggestion
```

File: packages/samples/vuejs/README.md
Comment (louis-bompart) [inline packages/samples/vuejs/README.md:1]: I think `patch` is nicer than `js` here for the md coloring

File: packages/samples/vuejs/cypress.config.ts [hunk 2]
```diff
@@ -3,10 +3,8 @@ import {defineConfig} from 'cypress';
 
 export default defineConfig({
   e2e: {
-    // We've imported your old cypress plugins here.
-    // You may want to clean this up later by importing these.
-    setupNodeEvents(on, config) {
-      return require('./cypress/plugins/index.js')(on, config);
```
Comment (louis-bompart) [inline packages/samples/vuejs/cypress.config.ts:9]: plugins is empty ➡️ 🗑️

File: packages/samples/vuejs/cypress/e2e/smoke-test.cypress.cy.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ describe('smoke test', {viewportHeight: 2000, viewportWidth: 2000}, () => {
       path: '**/rest/v15/analytics/*',
     }).as('analytics');
 
-    cy.visit('http://localhost:8080/').wait('@analytics');
+    cy.visit('http://localhost:5173/').wait('@analytics');
```
Comment (louis-bompart) [inline packages/samples/vuejs/cypress/e2e/smoke-test.cypress.cy.ts:18]: vite default port

File: packages/samples/vuejs/index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + Vue + TS</title>
```
Comment (louis-bompart) [inline packages/samples/vuejs/index.html:7]:
```suggestion
    <title>Vite + Vue + TS + Coveo Atomic</title>
```

**Review by olamothe — APPROVED** [review]

**Pull request merged**